cmd/farmer/piecestore-farmer.yaml

```diff
@@ -35,7 +35,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 60
       containers:
-      - image: "storjlabs/piecestore-farmer"
+      - image: "storjlabs/piecestore-storagenode"
```

Comment:
also the filename of this file. the image name here can probably just be storjlabs/storagenode

Reply:
I renamed the farmer folder in cmd/farmer/piecestore-farmer.yaml -> cmd/storagenode/piecestore-farmer.yaml & also the renamed 
piecestore-farmer.yaml -> storagenode.yaml